### ISSUE TITLE
form-has-label incorrectly fails when binding aria-label

### ISSUE THREAD

**Issue by calling** [maranran/eslint-plugin-vue-a11y#25]
**Describe the bug**
`v-bind:aria-label` should resolve errors caused by `form-has-label`

**To Reproduce**
1. Enable `form-has-label`
2. Create a bound input like so: `<input :aria-label="test" />`

**Expected behavior**
No error

**Screenshots**
`Each form element must have a programmatically associated label element  vue-a11y/form-has-label`

**Desktop (please complete the following information):**
 - OS: macOS
 - Node version: v12.13.0
- eslint-plugin-vue-a11y version: 0.0.31

**Comment by calling** [comment]
This seems to be addressed by https://github.com/maranran/eslint-plugin-vue-a11y/pull/21
